### DISCUSSION TITLE
New urls for French GEOPLATEFORME Service

### DISCUSSION THREAD

**Discussion by travelerdg** [mytourbook/mytourbook#1412] [General]
Since a couple of weeks, the French Geoportail service is deprecated. New urls are needed to access to the new French Geoplateforme service. Could you modify the content of the xml files ("Plan IGN V2" and "Photgraphies Aériennes HR") to take into account this change ?

Thanks a lot.

**Comment by FJBDev** [comment]
@travelerdg 
I have attached the updated map profile for IGN that contains the 3 map layers.

Note, you'll have to rename the `txt` extension to `xml` as GitHub prohibits files with XML extensions
[prof-geoportail-maps-2024-10-16.txt](https://github.com/user-attachments/files/17400897/prof-geoportail-maps-2024-10-16.txt)
